src/test/e2e/api/admin/feature-type.test.ts

```diff
@@ -1,6 +1,8 @@
 import dbInit from '../../helpers/database-init';
 import getLogger from '../../../fixtures/no-logger';
 import { setupApp } from '../../helpers/test-helper';
+import { validateSchema } from '../../../../lib/openapi/validate';
+import { featureTypesSchema } from '../../../../lib/openapi/spec/feature-types-schema';
```

Comment:
I think it could be nice to find a good way of preventing these deeply relative paths. Maybe using the same approach as the frontend?